common/v2/features/Dashboard/components/WalletBreakdown/WalletBreakdownView.tsx

```diff
@@ -416,6 +416,7 @@ const createOtherTokenAsset = (otherBalances: Balance[]) => ({
   ticker: translateRaw('WALLET_BREAKDOWN_OTHER_TICKER'),
   isOther: true,
   amount: 0,
+  uuid: EmptyUUID,
```

Comment:
This is used to have a standard object for the Other Tokens section of the balance breakdown stuff, so that we can treat it as a standard asset, despite it not being a standard asset.